<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="log_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oil_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>加油申请</string>
    </property>
   </widget>
   <widget class="QPushButton" name="error_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>故障报修</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="sig1">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/sig1.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sig2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/sig2.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sig3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/sig3.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="cloud">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/gps.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gps">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/cloud.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="power">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/new/prefix1/img/power.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="time_weather">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>541</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>541</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/new/prefix1/img/status_bg.png</pixmap>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>widget_2</zorder>
    <zorder>widget_3</zorder>
    <zorder>time_weather</zorder>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="driver">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>9</y>
       <width>71</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/new/prefix1/img/driver.jpg</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="widget_17" native="true">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>9</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>卡车编号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="car_num">
           <property name="text">
            <string>xxxx</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>驾驶员：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="driver_name">
           <property name="text">
            <string>xx</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="mask">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>71</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/new/prefix1/img/mask.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QLabel" name="backgroud">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>-1</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>131</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>车速：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="speed">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>52</y>
        <width>131</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>目的地：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dest">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>95</y>
        <width>137</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>剩余路程：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="leftroutie">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_10" native="true">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>9</y>
        <width>137</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>矿石种类：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="type">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_11" native="true">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>52</y>
        <width>137</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>当前载重：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="load">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_12" native="true">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>95</y>
        <width>131</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>油量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="oil">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_13" native="true">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>52</y>
        <width>113</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>矿石：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mine">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_14" native="true">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>95</y>
        <width>113</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>废石：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="trash">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="driver_video">
      <property name="geometry">
       <rect>
        <x>491</x>
        <y>9</y>
        <width>121</width>
        <height>123</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/new/prefix1/img/driver.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>总运输量：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="mask_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/new/prefix1/img/mask.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWebEngineView" name="geo_map" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_18" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget_19" native="true">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>装载</string>
        </property>
       </widget>
       <widget class="QLabel" name="led_r_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="led_g_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="input">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>input</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_20" native="true">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>卸载</string>
        </property>
       </widget>
       <widget class="QLabel" name="led_r_out">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="led_g_out">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="output">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>output</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="func">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>指令下发</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
